scripts/auto_sshfs_mount.py

```diff
@@ -57,11 +58,11 @@ def AttemptUnmount(local):
     
     #Check if the remote is already mounted, if so, unmount
     print(f'Attempting to unmount {local}...')
```

Comment:
print output has a tendency to get lost/redirected. Good to at least provide the option of writing to log file. For utility functions I've simply made `logger` and optional argument so that when called from POCS/huntsman-pocs classes you can just pass in `self.logger` and have the utility function write to the main log file.